docker-compose.yaml

```diff
@@ -43,4 +65,4 @@ services:
     volumes:
       - ./swagger.yaml:/swagger.yaml
     environment:
-      SWAGGER_JSON: /swagger.yaml
\ No newline at end of file
+      SWAGGER_JSON: /swagger.yaml
```

Comment:
Pushed the code again after adding the file, but it still isn't showing up here.

Reply:
If you plan to provide an API specification page, you'll need to expose the port so that anyone who wants to view the API spec can do so by accessing the host and port to see the API documentation